### DISCUSSION TITLE
PatchTool 7.10 11/14

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#483] [Announcements]
7.1相容性更新。

以下兩種方法擇一使用即可；請從release下方連結或內文連結下載檔案。

## 方法一：直接覆蓋遊戲檔案

解壓縮後直接覆蓋`SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\sqpack\ffxiv`內的同名檔案即可。內含三個000000開頭的字體相關檔案與三個0a0000開頭的文本相關檔案。

設定：Windows作業系統，非高清字體，以繁體中文覆蓋日文和英文，官方渠道版本。

設定相同的人可以考慮直接下載這包，（可能）比較不會遇到問題。其他作業系統未測試。

- ffxiv-20241114.rar：請從下方連結下載。

## 方法二：使用漢化器

使用方法請參照[readme](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/2.-%E4%BD%BF%E7%94%A8)。

檔案：

- FFXIVChnTextPatch-20241114.rar：漢化器本體。需要自行調整設定的人請下載這包手動漢化。請從下方連結下載。

視需求選用：

- [font-custom-20241005.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v3.3.1.0/font-custom-20241005.rar)：高畫質字體包，請取代原本的resource/font資料夾。請參閱[詳細使用說明](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。
- rawexd-zhcn-20241114.rar：簡體中文CSV，請取代原本resource/rawexd資料夾。
- backup-20241114.rar：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。只適用於這個release發布時的遊戲版本。

※若不替換字體會導致遊戲內無法正常顯示部分中文字。另外，目前大廳部分中文字必須使用高畫質字體包才能正常顯示。更多資訊可參閱[wiki頁面](https://github.com/GpointChen/FFXIVChnTextPatch-GP/wiki/6.-%E5%AD%97%E9%AB%94)。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v3.3.1.2'>PatchTool 7.10 11/14</a>.</em>

**Comment by atien369** [comment]
非常非常非常感恩大大的辛勞~

**Comment by Manbbb** [comment]
感謝GP大大，辛苦了

**Comment by JJ9013** [comment]
感恩感恩

**Comment by yamu0921** [comment]
感謝大大！！！

**Comment by kernelogic** [comment]
非常感谢，如此迅速！

**Comment by a61456711** [comment]
太迅速了(❁´◡`❁)

**Comment by hanyang08** [comment]
非常謝謝! 理解劇情不可或缺

**Comment by Bos-hunting** [comment]
請問會有7.11的兼容版嗎

**Reply by kalispp** [reply]
此版本可兼容7.11，我弄起來沒甚麼太大問題，有一些小的顯示問題(不太影響主要功能)要等作者身體康復再看看了
